app/main.py

```diff
@@ -2,24 +2,84 @@
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: type, name: str) -> None:
+        self.name = name
+
+    def __get__(self,
+                instance: "SlideLimitationValidator",
+                owner: type
+                ) -> tuple:
+        return getattr(instance, self.name)
+
+    def __set__(self,
+                instance: "SlideLimitationValidator",
+                value: int
+                ) -> None | bool:
+        if not (self.min_amount <= value <= self.max_amount):
+            raise ValueError
+        return True
 
 
 class Visitor:
-   pass
+    def __init__(self, name: str, age: int, weight: int, height: int) -> None:
+        self.name = name
+        self.age = age
+        self.weight = weight
+        self.height = height
 
 
 class SlideLimitationValidator(ABC):
-    pass
+    def __init__(self,
+                 age: IntegerRange,
+                 weight: IntegerRange,
+                 height: IntegerRange
+                 ) -> None:
+        self.age = age
+        self.weight = weight
+        self.height = height
 
 
 class ChildrenSlideLimitationValidator(SlideLimitationValidator):
-    pass
+    age = IntegerRange(4, 14)
+    weight = IntegerRange(20, 50)
+    height = IntegerRange(80, 120)
+
+    def __init__(self,
+                 age: IntegerRange,
+                 weight: IntegerRange,
+                 height: IntegerRange
+                 ) -> None:
+        super().__init__(age, weight, height)
```

Comment:
Delete this `__init__` here and in the `AdultSlideLimitationValidator` class as well, you could rely on the parent class implementation.